yarn-project/end-to-end/src/e2e_block_building.test.ts

```diff
@@ -420,13 +418,27 @@ describe('e2e_block_building', () => {
     afterAll(() => teardown());
 
     it('calls a method with nested encrypted logs', async () => {
-      const thisWallet = new AccountWalletWithSecretKey(pxe, ownerWallet, owner.secret, owner.salt);
       const address = owner.address;
 
+      const values = {
```

Comment:
The expected value changed to a struct because that's how the event looks and the getPrivateEvents endpoint returns the value ABI decoded and not as an array of fields.